src/core/core.datasetController.js

```diff
@@ -566,9 +579,8 @@ export default class DatasetController {
 	 * @returns {object} parsed item - item containing index and a parsed value
 	 * for each scale id. _custom is optional
 	 * Example: {xScale0: 0, yScale0: 1, _custom: {r: 10, foo: 'bar'}}
-	 * @private
 	 */
-	_parseObjectData(meta, data, start, count) {
+	parseObjectData(meta, data, start, count) {
```

Comment:
I would have kept most of these parsing methods as `@protected`

Reply:
Done